From 9c5af2998d9e04cda1670dd3dc772bfcfb5c731f Mon Sep 17 00:00:00 2001
From: "wenxin.leong" <wenxin.leong@infineon.com>
Date: Tue, 16 Nov 2021 17:04:45 +0800
Subject: [PATCH] integrate TPM into AOSP (part 2)

---
 Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.bp b/Android.bp
index a7f1a27704..fc348e126b 100755
--- a/Android.bp
+++ b/Android.bp
@@ -31,7 +31,7 @@
 
 cc_prebuilt_library_shared {
     name: "openssl-libcrypto",
-    //vendor: true,
+    vendor: true,
     //host_supported: true,
     compile_multilib: "first",
     export_include_dirs: [
@@ -47,7 +47,7 @@ cc_prebuilt_library_shared {
 
 cc_prebuilt_library_shared {
     name: "openssl-libssl",
-    //vendor: true,
+    vendor: true,
     //host_supported: true,
     compile_multilib: "first",
     export_include_dirs: [
-- 
2.25.1

